README.md

```diff
@@ -8,7 +8,10 @@
 [![FlakeHub](https://img.shields.io/endpoint?url=https://flakehub.com/f/stackbuilders/nixpkgs-terraform/badge)](https://flakehub.com/flake/stackbuilders/nixpkgs-terraform)
 [![flakestry.dev](https://flakestry.dev/api/badge/flake/github/stackbuilders/nixpkgs-terraform)](https://flakestry.dev/flake/github/stackbuilders/nixpkgs-terraform)
 
-A collection of Terraform versions that are automatically updated.
+This [flake](https://nixos.wiki/wiki/Flakes) exposes a collection of Terraform
```

Comment:
@jsl there is already a [Usage](https://github.com/stackbuilders/nixpkgs-terraform?tab=readme-ov-file#usage) section that explains how to use this project; however, I expanded the introduction a little bit to provide a better overview of this project. Please let me know your thoughts.